gluten-core/src/main/resources/substrait/proto/substrait/algebra.proto

```diff
@@ -444,6 +444,7 @@ message Rel {
     ExpandRel expand = 15;
     WindowRel window = 16;
     GenerateRel generate = 17;
+    WriteRel write = 18;
```

Comment:
Please document this change to Substarit in https://github.com/oap-project/gluten/blob/main/docs/developers/SubstraitModifications.md.